src/components/Contacts/ContactListTable.jsx

```diff
@@ -107,11 +115,12 @@ const ContactListTable = ({ contacts, deleteContact }) => {
         columns={columnTitlesArray}
         rows={contacts?.map((contact) => ({
           id: contact.webId,
-          'First Name': contact.givenName,
-          'Last Name': contact.familyName,
-          Profile: contact,
-          Message: contact,
-          Delete: contact
+          'First Name': contact.givenName || '',
+          'Last Name': contact.familyName || '',
+          webId: contact.webId,
+          Profile: contact, // Ensure this is rendered as intended
+          Message: contact, // Verify handleSendMessage uses correct data
+          Delete: contact // Check deleteContact logic
```

Comment:
The extra comments could be removed. If need to be kept, they should be on the logic and functions that processes `contact`